### PR TITLE
Added the on/off option for the feature for C# and VB - Inline Parameter Name Hints

### PR DESCRIPTION
**Pull request by akhera99**
Hello,

Creating a pull request to be able to merge adding the on/off option feature to the existing Inline Parameter Name Hints branch.
The option is located at Tools->Options->Text Editor->Basic/C#->Advanced under the 'Editor Help' section. It is initially turned off for everyone and is marked with 'experimental' so everyone knows it's not a full feature.

C# Screenshot:
image

VB Screenshot:
image

The alt key and 'l' are also tied to the checkbox.

Please let me know if anyone has any feedback!

### CODE REVIEW

File: src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml.cs [hunk 1]
```diff
@@ -67,6 +67,8 @@ public AdvancedOptionPageControl(OptionStore optionStore, IComponentModel compon
             BindToOption(EnableHighlightKeywords, FeatureOnOffOptions.KeywordHighlighting, LanguageNames.CSharp);
             BindToOption(RenameTrackingPreview, FeatureOnOffOptions.RenameTrackingPreview, LanguageNames.CSharp);
 
+            BindToOption(DisplayInlineParameterNameHints, FeatureOnOffOptions.InlineParameterNameHints, LanguageNames.CSharp);
```
Comment (jasonmalinowski) [inline src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml.cs:70]: Can we keep this roughly sorted in the same order as the dialog actually is?

File: src/VisualStudio/VisualBasic/Impl/Options/AdvancedOptionPageControl.xaml.vb [hunk 2]
```diff
@@ -82,6 +83,7 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.Options
             BindToOption(Show_completion_list, RegularExpressionsOptions.ProvideRegexCompletions, LanguageNames.VisualBasic)
 
             BindToOption(Editor_color_scheme, ColorSchemeOptions.ColorScheme)
+
```
Comment (jasonmalinowski) [inline src/VisualStudio/VisualBasic/Impl/Options/AdvancedOptionPageControl.xaml.vb:86]: No reason for extra line.

File: src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml.cs (hunk 1 above)
Comment (akhera99) [reply src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml.cs:70]: Fixed

File: src/VisualStudio/VisualBasic/Impl/Options/AdvancedOptionPageControl.xaml.vb (hunk 2 above)
Comment (akhera99) [reply src/VisualStudio/VisualBasic/Impl/Options/AdvancedOptionPageControl.xaml.vb]: Removed

**Pull request merged**